CONTRIBUTING.md

```diff
@@ -112,7 +112,7 @@ If none of these let you reproduce, there might be some race condition
 or continuous integration breakage; please file a bug.
 
 **Running tests locally.**
-To run tests locally with `new-build`, you will need to know the
+To run tests locally with `v2-build`, you will need to know the
```

Comment:
should here be double ticks

Reply:
I'll use double ticks here. But should I revert the new -> v2 changes in this doc to avoid unnecessary changes?

Reply:
No, using `v2-` is better.

Reply:
Ok, keeping v2. Also a I realized this is markdown file, so single ticks should be the way to go.